src/OrbitGl/CaptureViewElementTest.cpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include "CaptureViewElementTester.h"
+
+namespace orbit_gl {
+
+class UnitTestCaptureViewElement : public CaptureViewElement {
```

Comment:
Nit: How about `MockCaptureViewElement`?

Reply:
Also, I'm a little bit curious why we are not using gmock to make the test?

Reply:
I think mock is the wrong term here, and I don't benefit from gmock here at all. I'm not great with testing terminology, I'm not sure what this would correspond to - a mock usually verifies that some other object (the object you want to test) makes the correct calls, but in this case I really only want to test CaptureViewElement itself. Probably a fake is the closest here?